### ISSUE TITLE
Testing Framework

### ISSUE THREAD

**Issue by rphmeier** [paritytech/contracts#93]
cc @tomusdrw @lght 

Is Truffle good here? We just need something we can use in the CI to ensure tests are passing. I know you talked about a Rust-based testing framework before.

**Comment by tomusdrw** [comment]
Yeah, we had some ideas. Should have something to show next week.
